server/aws-lsp-codewhisperer/src/shared/amazonQServiceManager/configurationUtils.ts

```diff
@@ -67,10 +67,15 @@ interface QInlineSuggestionsConfig {
     extraContext: string | undefined // aws.q.inlineSuggestions.extraContext
 }
 
+interface QProjectContextConfig {
+    enableIndexing: boolean // aws.q.projectContext.enableIndexing
```

Comment:
nit: should this flag or section be named specific to client-side project context? When we add server-side project context - that may need settings too.

Reply:
Good call - qualified it: `enableLocalIndexing`